src/components/BulkEnrollmentResultsDownloadPage/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Redirect, useParams } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { logError } from '@edx/frontend-platform/logging';
+import { camelCaseObject } from '@edx/frontend-platform';
+import { ToastsContext } from '../Toasts';
+import LicenseManagerApiService from '../../data/services/LicenseManagerAPIService';
+
+const BulkEnrollmentResultsDownloadPage = ({ enterpriseId }) => {
+  const { enterpriseSlug, bulkEnrollmentJobId } = useParams();
+  const { addToast } = useContext(ToastsContext);
+  const [isLoading, setLoading] = useState(true);
+  const [redirectUrl, setRedirectUrl] = useState(null);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    if (enterpriseId && isLoading) {
```

Comment:
nit: Is `enterpriseId` always truthy since it's a required prop, so doesn't really need to be in the conditional? Or the reverse, is there a prop type warning in the console that says `enterpriseId` isn't provided at some point in the rendering cycle and it shouldn't be a required prop?

Reply:
👍